benchmarks/load_test/load_test.py

```diff
@@ -0,0 +1,109 @@
+import os
+import requests
+import subprocess
+import yaml
+from collections import namedtuple
+
+import numpy as np
+import jinja2
+import click
+from locust import HttpLocust
+
+TEMPLATE_FILE = "task_set.py.jinja2"
+Action = namedtuple("Action", "name path json")
```

Comment:
I guess I probably wrote this originally, but do you think it should be named `Task` instead, isn't that what locust refers to these things as? :man_shrugging: